pybamm/solvers/c_solvers/idaklu/CasadiSolverOpenMP.hpp

```diff
@@ -142,6 +145,16 @@ class CasadiSolverOpenMP : public CasadiSolver
    * @brief Allocate memory for matrices (noting appropriate matrix format/types)
    */
   void SetMatrix();
+
+  /**
+   * @brief Apply user-configurable IDA options
+   */
+  void SetSolverOptions();
+
+  /**
+   * @brief Check the return flag for errors
+   */
+  void CheckErrors(int& flag);
```

Comment:
pairs with the previous change
```suggestion
  void CheckErrors(int const & flag);
```